### PR TITLE
feat(ChangePlan): add compare features

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add without BE and simulate have the data of the plans.

image

image

### CODE REVIEW

File: src/components/ChangePlan/ChangePlan.js [hunk 1]
```diff
@@ -1,35 +1,62 @@
...
-    descriptionPlan: _('change_plan.until_x_subscribers', { subscribers: planQuantity }),
-    action: {
-      url: planUrl,
-      text: _('change_plan.calculate_price'),
-    },
+  const [isFeaturesVisible, setIsFeaturesVisible] = useState(false);
+  // Allow preselect by email or contacts
+  const selectedType = extractParameter(location, queryString.parse, 'selected-type') || '';
+  const currentPlan = getPlanById(planId);
+  const planPlus = getPlanPlusByPlanId(planId);
```
Comment (cbernat) [inline src/components/ChangePlan/ChangePlan.js:54]: Couldn´t you perhaps add both plans into an array so then you can map both plans into the same html? The one for the cards

Comment (fcoronelmakingsense) [reply src/components/ChangePlan/ChangePlan.js]: ready

File: src/i18n/en.js [hunk 2]
```diff
@@ -34,12 +34,20 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
 
 export default {
   change_plan: {
+    all_of_plan: `Everything from {plan} plus:`,
```
Comment (cbernat) [inline src/i18n/en.js:37]:
```suggestion
    all_of_plan: `Everything from {plan} Plus:`,
```

File: src/i18n/en.js [hunk 3]
```diff
@@ -34,12 +34,20 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
...
+    compare_features: 'Comparare Features',
     description: 'Do you send periodically or one newsletter per month?',
+    email_parameter: 'Email parameter',
+    features: 'Features:',
     per_month: 'per month',
     recommended: 'Recommended',
     selected_type: 'Plans selected by: ',
+    shipping_limit: 'Shipping limit',
     since: 'Since',
+    site_tracking: 'Site tracking',
```
Comment (cbernat) [inline src/i18n/en.js:49]:
```suggestion
    site_tracking: 'Site Tracking',
```

File: src/i18n/en.js [hunk 4]
```diff
@@ -34,12 +34,20 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
...
     calculate_price: 'Calculate Price',
+    cancel_campaign: 'Cancel Campaign',
+    compare_features: 'Comparare Features',
     description: 'Do you send periodically or one newsletter per month?',
+    email_parameter: 'Email parameter',
+    features: 'Features:',
     per_month: 'per month',
     recommended: 'Recommended',
     selected_type: 'Plans selected by: ',
+    shipping_limit: 'Shipping limit',
```
Comment (cbernat) [inline src/i18n/en.js:47]:
```suggestion
    shipping_limit: 'Shipping Limit',
```

File: src/components/ChangePlan/ChangePlan.js [hunk 5]
```diff
@@ -51,27 +79,69 @@ const ChangePlan = ({ location }) => {
...
+                  ) : null}
+                </Card>
+              ))}
             </div>
           </div>
+          <div className="dp-align-center p-t-30 p-b-30">
+            <button
+              className="dp-compare-details-plans"
+              onClick={() => {
+                showFeatures();
```
Comment (cbernat) [inline src/components/ChangePlan/ChangePlan.js:139]: Why not this?
```suggestion
                setIsFeaturesVisible(!isFeaturesVisible);
```

File: src/i18n/en.js (hunk 3 above)
Comment (cbernat) [reply src/i18n/en.js:49]: image

File: src/i18n/es.js [hunk 6]
```diff
@@ -34,12 +34,20 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
 
 export default {
   change_plan: {
+    all_of_plan: `Todo lo de {plan} más:`,
     calculate_price: 'Calcular Precio',
+    cancel_campaign: 'Cancelar Campaña',
+    compare_features: 'Comparar Funcionalidades',
     description: '¿Realizas envíos periodicamente o un newsletter al mes?',
+    email_parameter: 'Parametros de email',
```
Comment (cbernat) [inline src/i18n/es.js:42]:
```suggestion
    email_parameter: 'Parámetros de Email',
```

File: src/i18n/es.js [hunk 7]
```diff
@@ -34,12 +34,20 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
...
+    features: 'Funcionalidades:',
     per_month: 'por mes',
     recommended: 'Recomendado',
     selected_type: 'Planes por: ',
+    shipping_limit: 'Límite de envíos',
     since: 'Desde',
+    site_tracking: 'Comportamiento en Sitio',
+    smart_campaigns: 'Campañas inteligentes',
```
Comment (cbernat) [inline src/i18n/es.js:50]:
```suggestion
    smart_campaigns: 'Envíos Inteligentes',
```
image

File: src/i18n/en.js [hunk 8]
```diff
@@ -34,12 +34,20 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
...
     since: 'Since',
+    site_tracking: 'Site tracking',
+    smart_campaigns: 'Smart Campaigns',
```
Comment (cbernat) [inline src/i18n/en.js:50]:
```suggestion
    smart_campaigns: 'Smart Send Out',
```
image

**Comment by cbernat** [comment]
Some minor content changes the rest seems fine! 👍

File: src/i18n/es.js [hunk 9]
```diff
@@ -34,12 +34,20 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
...
     calculate_price: 'Calcular Precio',
+    cancel_campaign: 'Cancelar Campaña',
+    compare_features: 'Comparar Funcionalidades',
     description: '¿Realizas envíos periodicamente o un newsletter al mes?',
+    email_parameter: 'Parámetros de Email',
+    features: 'Funcionalidades:',
     per_month: 'por mes',
     recommended: 'Recomendado',
     selected_type: 'Planes por: ',
+    shipping_limit: 'Límite de envíos',
```
Comment (cbernat) [inline src/i18n/es.js:47]:
```suggestion
    shipping_limit: 'Límite de Envíos',
```
This is for consistency with English version, nothing more.

**Pull request merged**